package.json

```diff
@@ -52,7 +52,7 @@
         "build": "node scripts/update",
         "clean": "rimraf .nyc_output coverage",
         "coverage": "opener ./coverage/lcov-report/index.html",
-        "format": "prettier --loglevel warn --write \"**/*.{js,css,md}\"",
+        "format": "prettier --write \"**/*.js\"",
```

Comment:
Do we not want the `.md` files here still?

Also, we may want to add `.json` files 🤔

Reply:
it was removed as prettier does not play well with `markdownlint`.😅

yes, I'll add `json`. 👍